rpc/errors.go

```diff
@@ -104,10 +104,10 @@ func (e *invalidParamsError) ErrorCode() int { return -32602 }
 func (e *invalidParamsError) Error() string { return e.message }
 
 type CustomError struct {
-	Code    int
-	Message string
+	Code            int
+	ValidationError string
```

Comment:
This error type is kind of misplaced in package rpc. I'm merging anyway, but please move it in the next iteration.